lib/cisco_node_utils/cmd_ref/aaa_auth_login_service.yaml

```diff
@@ -1,22 +1,22 @@
 # aaa_auth_login_service
```

Comment:
Haven't checked the impacted yaml files but all looks good and big improvement. :+1: